src/ServiceManagement/PublishSettingsImporter.cs

```diff
@@ -91,7 +91,7 @@ private static PublishData DeserializePublishData(Stream stream)
                 certificateString = profile.ManagementCertificate;
             }
 
-            X509Certificate2 certificate = new X509Certificate2(Convert.FromBase64String(certificateString), string.Empty);
+            X509Certificate2 certificate = new X509Certificate2(Convert.FromBase64String(certificateString), string.Empty, X509KeyStorageFlags.PersistKeySet);
```

Comment:
I think we can do this in a more targeted way by making the change here:https://github.com/Azure/azure-powershell-common/blob/master/src/Authentication.Abstractions/DiskDataStore.cs#L223

This would involve only one assembly, which is guaranteed to be backwards compatible, where this one is not.

The idea would be to make a change like this before the certificate is stored:

``` c#
X509Certificate2 storedCertificate = new X509Certificate2(certificate.RawData, null, X509KeyStorageFlags.PersistKeySet);
```

Then store this certificate created from the other.  This would require a test to make sure it resolved the problem, however, the reason to favor this change is because this library is *guaranteed* to be backwards compatible, and only one dll would need to change.